bin/knit_lessons.sh

```diff
@@ -4,5 +4,5 @@
 # The Makefile passes in the names of files.
 
 if [ $# -ne 0 ] ; then
-    Rscript -e "source('bin/generate_md_episodes.R')"
+    Rscript -e "source('bin/generate_md_episodes.R')" $*
```

Comment:
I'd suggest using `"$@"` or `"$*"` instead of `$*`.

Reply:
thanks for the suggestion. I pushed a commit to use "$@"